docs/commands/lint.md

```diff
@@ -1,77 +1,121 @@
 # `lint`
 
-Redocly OpenAPI CLI can identify and report on problems found in OpenAPI definitions, with the goal of avoiding bugs and making APIs more consistent. The `lint` command reports on problems and executes preprocessors and rules. Unlike the `bundle` command, `lint` doesn't execute decorators.
+## Introduction
 
-Use custom rules, plugins, preprocessors, and the built-in rules to define your API design standards. Every rule is a plugin that can be added at runtime.
+Redocly OpenAPI CLI can identify and report on problems found in OpenAPI definitions. This allows you to avoid bugs and make API definitions more consistent.
 
+The `lint` command reports on problems and executes preprocessors and rules. Unlike the `bundle` command, `lint` doesn't execute decorators.
 
-```mermaid
-graph LR
-    preprocessors[Preprocessors]
-    rules[Rules]
-    preprocessors --> rules
+:::success Tip
+To learn more about preprocessors, rules, and decorators, please refer to [this](../custom-rules.md) document.
+:::
+
+## Usage
+
+```bash
+openapi lint
+openapi lint <entrypoints>...
+openapi lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
+openapi lint [--generate-ignore-file] [--help]
+openapi lint --version
 ```
 
+## Options
+
+option                   | type      | required?    | default     | description
+-------------------------|:---------:|:------------:|:-----------:|------------
+`entrypoints`            | `array`   | no           | `[]`        | Array of API definition filenames that need to be linted. (See [the section below](#entrypoints) for more options)
+`--config`               | `string`  | no           | -           | Specify path to the config file
```

Comment:
It looks like a lot of the same feedback I left for [the bundle command](https://github.com/Redocly/openapi-cli/pull/350) applies here as well. Instead of me spamming you with individual comments and suggestions, you can just model this `lint` page on the `bundle` page and apply the same principles suggested in that other PR. Then I'll do another review just to cover any loose ends.

Reply:
@skadinna I made another stage of review. Seems like everything is OK. Please proceed with your review